Apps/VisionOS/VisionOS/VisionOSApp.swift

```diff
@@ -3,9 +3,11 @@ import SwiftUI
 @main
 struct VisionOSApp: App {
     @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+    let viewModel = ViewModel()
     var body: some Scene {
         WindowGroup {
             MainScreen()
+                .environmentObject(viewModel)
```

Comment:
When I see "ViewModel" in the name, I think of the design pattern where there is 1 ViewModel that is tightly coupled to 1 View in the app. Here, this code is using a ViewModel that is shared across the whole app. 

I think that the ViewModel here is being used to share a common state with all the screens of the app. If that's true, would a better name to "AppState"? Maybe even "AppViewModel" to differentiate between an app wide model vs a model for 1 screen?